scout/adapter/mongo/case.py

```diff
@@ -686,18 +697,17 @@ def update_dynamic_gene_list(
         )
         return updated_case
 
-    def case(self, case_id=None, institute_id=None, display_name=None):
+    def case(
+        self,
+        case_id: str = None,
+        institute_id: str = None,
+        display_name: str = None,
+        projection: Dict = None,
```

Comment:
Very good idea!

Reply:
Good call; we haven't had to worry much about memory on the web server, only db server before, but it is definitely becoming an issue. Very timely!